### PR TITLE
Set up cloudinary

### PR DESCRIPTION
**Pull request by garzeah**
### What this PR does (required):
- Set up cloudinary

### Screenshots / Videos (required):
- Not applicable

### Any information needed to test this feature (required):
- Can test a method from the api as long as you have valid credentials

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -1,82 +1,89 @@
...
+const appointmentRouter = require('./routes/appointment');
+const meetingRouter = require('./routes/meeting');
 
 const { json, urlencoded } = express;
 
 connectDB();
 const app = express();
 const server = http.createServer(app);
 
+// Initializes our cloudinary
```
Comment (ajaytomgeorge) [inline server/app.js]: This obvious, you can remove the comment

File: server/app.js [hunk 2]
```diff
@@ -1,82 +1,89 @@
...
+	api_secret: process.env.CLOUDINARY_API_SECRET,
+	secure: true
+});
+
 const io = socketio(server, {
-  cors: {
-    origin: "*",
-  },
+	cors: {
+		origin: '*'
```
Comment (ajaytomgeorge) [inline server/app.js]: Formatting broken, please reformat

File: server/app.js [hunk 3]
```diff
@@ -1,82 +1,89 @@
...
-io.on("connection", (socket) => {
-  console.log("connected");
+io.on('connection', (socket) => {
+	console.log('connected');
 });
 
-if (process.env.NODE_ENV === "development") {
-  app.use(logger("dev"));
+if (process.env.NODE_ENV === 'development') {
+	app.use(logger('dev'));
```
Comment (ajaytomgeorge) [inline server/app.js]: Here too

File: server/app.js [hunk 4]
```diff
@@ -1,82 +1,89 @@
...
-  },
+	cors: {
+		origin: '*'
+	}
 });
 
-io.on("connection", (socket) => {
-  console.log("connected");
+io.on('connection', (socket) => {
+	console.log('connected');
```
Comment (ajaytomgeorge) [inline server/app.js]: Here too

File: server/app.js [hunk 5]
```diff
@@ -1,82 +1,89 @@
...
 app.use(session({ secret: process.env.SESSION_SECRET }));
 app.use(passport.initialize());
 app.use(passport.session());
-app.use(express.static(join(__dirname, "public")));
+app.use(express.static(join(__dirname, 'public')));
 
 app.use((req, res, next) => {
-  req.io = io;
-  next();
+	req.io = io;
```
Comment (ajaytomgeorge) [inline server/app.js]: here

File: server/app.js [hunk 6]
```diff
@@ -1,82 +1,89 @@
...
+const meetingRouter = require('./routes/meeting');
 
 const { json, urlencoded } = express;
 
 connectDB();
 const app = express();
 const server = http.createServer(app);
 
+// Initializes our cloudinary
+cloudinary.config({
```
Comment (ajaytomgeorge) [inline server/app.js:25]: Setting up involves some more elements, you can see this example [https://cloudinary.com/blog/node_js_file_upload_to_a_local_server_or_to_the_cloud](example)

Comment (bonnieli) [reply server/app.js:25]: I think this set up should be fine, would you be able to point out what is currently be missing? 

I assume @garzeah followed this guide instead: https://cloudinary.com/documentation/node_integration

**Review by bonnieli — COMMENTED** [review]
And just to summarize what Ajay suggested - looks like a bunch of linting changes. This is fine but I would sync with the team that this is the standard and same linting config is standard across the team :)

File: server/app.js (hunk 1 above)
Comment (garzeah) [reply server/app.js]: Removed!

File: server/app.js (hunk 5 above)
Comment (garzeah) [reply server/app.js:56]: Hi, do you have any clues as to how to fix this? On my client it shows that I have 2 spaces but when I push it gets messed up.

File: server/app.js (hunk 6 above)
Comment (ajaytomgeorge) [reply server/app.js:25]: @bonnieli Yeah.. this is set up should be fine. Please resolve the conflicts though :)

Comment (garzeah) [reply server/app.js:25]: Resolved, thanks!

**Review by moffatethan — APPROVED** [review]
Good to merge!

**Pull request merged**